### ISSUE TITLE
Software review – inclusion of bspwm in Fedora repositories

### ISSUE THREAD

**Issue by Lesik** [baskerville/bspwm#413]
Hey,

we are currently in the process of including bspwm into the Fedora repositories. While reviewing the software, we have found some issues in the latest (0.9) release:

1. The `bspwm.desktop` file does comply with the [freedesktop specifications](https://specifications.freedesktop.org/desktop-entry-spec/latest/ar01s05.html).
This was fixed in https://github.com/baskerville/bspwm/commit/2b5cf373dd3c96dc0c5c3b44fef911ab47f2712a (see my comment for more info).

2. There is an issue when compiling with Fedora-specific CFLAGS. Actually, the issue persists when appending any CFLAGS at all, even if they're empty. I haven't looked into what exactly causes this, because I have found that this is already fixed in the newest git checkout. FYI, the output was the following:
```
cc -O2 -g -pipe -Wall -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector-strong --param=ssp-buffer-size=4 -grecord-gcc-switches -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -m64 -mtune=generic  -c -o bspwm.o bspwm.c
bspwm.c: In function ‘main’:
bspwm.c:71:20: error: ‘VERSION’ undeclared (first use in this function)
     printf("%s\n", VERSION);
                    ^
bspwm.c:71:20: note: each undeclared identifier is reported only once for each function it appears in
Makefile:37: recipe for target 'bspwm.o' failed
make: *** [bspwm.o] Error 1
```

Since I would rather not package software from the git branch because it is not known how stable the software is at the time of my checkout, I would like to ask you to release a minor update which
1. complies with the freedesktop specifications and
2. compiles correctly when specifying CFLAGS.
This would allow us to complete the packaging process, deliver bspwm to Fedora users (as well as RHEL, CentOS and SL users later-on) and spread the word about this great piece of software.

**Comment by baskerville** [comment]
Thanks, I've just released version *0.9.1*.

**Comment by Lesik** [comment]
I am pleased to announce that bspwm will be available in the official repositories of Fedora 25 and higher.

https://apps.fedoraproject.org/packages/bspwm

Thank you.
